package.json

```diff
@@ -23,5 +23,16 @@
   },
   "bugs": {
     "url": "https://github.com/Pomax/react-onoutsideevent/issues"
+  },
+  "devDependencies": {
+    "envify": "^3.4.0",
+    "gulp": "^3.9.0",
+    "gulp-rename": "^1.2.2",
+    "gulp-uglify": "^1.4.2",
+    "vinyl-transform": "^1.0.0"
+  },
+  "scripts": {
+    "build": "./node_modules/.bin/gulp",
```

Comment:
npm scripts already run with `node_modules/.bin` as implicit PATH, so you can just say `gulp` here.